float64[] target
---
bool target_reached
float64[] target_result
